platform/how-to/uninstall.mdx

```diff
@@ -2,40 +2,95 @@
 title: Uninstall vCluster Platform
 sidebar_label: Uninstall vCluster Platform
 sidebar_position: 8
+description: Learn how to uninstall the vCluster Platform from your Kubernetes cluster using the vCluster CLI or Helm.
 ---
 
-## Delete Helm Release
+import VersionBadge from "@site/src/components/VersionBadge";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
 
-```bash
-helm delete -n vcluster-platform loft
+import Prerequisites from '../_partials/install/base-prerequisites.mdx';
+
+
+This guide provides instructions on how to uninstall the platform from your Kubernetes cluster.
+
+:::warning
+This action is performed against the cluster the kube-context is pointing to, not the platform instance that is logged in. It does not require logging in to the platform.
+:::
+
+## Prerequisites
+
+<Prerequisites />
+
+## Uninstall
+
+There are two ways to uninstall the platform.
+
+<Tabs>
+    <TabItem value="Using vCluster CLI">
+<VersionBadge version="v4.2" vclusterVersion="v0.22" />
```

Comment:
I don't know if the platform matters but more what vCluster version is as that's where the command is. I would double check with @rohantmp though.

Reply:
I'm changing it to only say vclusterVersion for now, we can change later.